Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings

```diff
@@ -2851,6 +2899,12 @@
 /* No comment provided by engineer. */
 "shortcuts.shortcutSettingsOpenPlaylistTitle" = "Oynatma Listesi Aç";
 
+/* Description of Open Tabs from Other Devices Shortcut in Settings Screen */
+"shortcuts.shortcutSettingsOpenSyncedTabsDescription" = "Sesli Yardımcı Siri aracılığıyla Sekmeleri Diğer Cihazlardan açmak için Kısayolları kullanın";
```

Comment:
"Sesli Yardımcı Siri aracılığıyla Diğer Cihazlardaki Sekmeleri açmak için Kısayolları kullanın"